tests/__tests__/pastel/Pastel_test.re

```diff
@@ -61,7 +171,7 @@ describe("Pastel", ({describe, test}) => {
               );
             });
           });
-          describe("Pastel.length fuzz testing", ({test}) => {
+          describeSkip("Pastel.length fuzz testing", ({test}) => {
```

Comment:
Should this still be skipped?

Reply:
Definitely not, I'll try and see why this doesn't fail CI?